introduction.md

```diff
@@ -6,27 +6,120 @@ title: Introduction
+* **Bookmarks**: Saved host search statements as bookmarks for repetitive use.
+* **Web User Interface**: Powerful web UI built on modern technologies.
+* **API/CLI**: Powerful API, whole infrastructure can be managed via external tools.
+* **Community powered**: Foreman ships with many configuration and remote execution templates maintained by the community.
+
+**Inventory**
+
+* **Hosts inventory**: Inventory of managed servers (nodes).
+* **Host groups**: Host grouping with common options, parameters and support for field inheritance.
+* **NIC discovery**: Automatic creation of network interfaces (regular, bond, bridge, VLAN), Operating System and Architecture (according to facts reported by hosts).
+* **Common search**: Powerful search across whole application with smart completion.
+* **Subnet & Domain inventory**: Manage any number of networks via Foreman Proxy DHCP & DNS modules (including VLANs).
+* **IPAM**: Manage DHCP reservations on various providers like ISC DHCP, MS DHCP or Infoblox, free IP addresses can be allocated on the fly or via Foreman database.
+* **DNS and identity management**: DNS or realm entries can be automatically created for each host in Foreman inventory.
+
+**Provisioning**
+
+* **Manage PXE**: Foreman provides full management of PXE configuration of PXELinux, Grub, Grub2 and iPXE for maximum network boot flexibility.
+* **Install OS**: Initiate unattended provisioning of various Operating Systems via extensive set of templates and snippets maintained by the community.
+* **Build VMs**: Integrate with hypervisors like VMWare vCenter, Red Hat Enterprise Virtualization, oVirt or libvirt to create instances directly from Foreman UI/API/CLI either from images or via PXE.
+* **Create cloud instances**: Integrate with clouds like OpenStack, Rackspace, Amazon EC2 or Google Compute Engine directly from Foreman UI/API/CLI.
+* **Host network configuration**: Provisioning templates which create network configuration for installed hosts including bonding, bridging and VLAN trunk support.
+* **Configuration management bootstrap**: Template snippets for bootstrapping initial configuration of configuration management software including signing client keys with CA.
+* **IPv6**: Foreman can manage IPv6 addresses on non-provisioning interfaces (PXE provisioning on IPv6 is [work in progress](link to redmine bug goes here))
+* **Templating engine**: Templates based on ERB for OS installation recipes (Kickstart, Preseed), jobs (SSH scripts, Ansible jobs), partitioning schemes and other types.
+* **Compute Resources**: Modules or plugins for integration with hypervisors and cloud infrastructure.
+* **Compute Profiles**: Common compute profiles across multiple clouds or virtualization (e.g. xxsmall, large, medium).
+
+**Server discovery**
+
+* **Host discovery**: Boot unknown hardware from network or via local media (USB stick) and let it register to Foreman for automatic, semi-automatic or fully manual provisioning.
```

Comment:
I won't call it manual provisioning (as manual is pressing next-next over ilo connection), maybe something like workflow automated provisioning vs on demand (e.g. user action)

Reply:
OK